CONTRIBUTING.md

```diff
@@ -102,6 +102,8 @@ accordingly in the steps below.)
    `git checkout -b fix_alabala`
    - Don't forget to keep formatting standards, run `v fmt -w YOUR_MODIFIED_FILES`
      before committing (if you have not run the commands from 3.1)
+   - If you changed Markdown (`.md`) files, check them `v check-md YOUR_MODIFIED_FILES`
+     before committing (if you have not run the commands from 3.1)
```

Comment:
```suggestion
     before committing.
```

The precommit hook does `v fmt -w modified.v` for all .v files. It will not affect .md files.